### PR TITLE
Show quick links to more bundle discount eligible products when adding them would help

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Show quick links to eligible products for bundle discounts in cart page

There are 3 categories: Home, Car, Pet.  We exclude those represented in the cart and show links to others if session is eligible for bundle discount experiment

Category config is very quick and dirty for the purpose of this experiment. Product IDs, categories, their mapping and titles all being hardcoded.  We're using pillow images and links from GraphQL API, since those are already available without API change

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
* **#4278** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4278?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/CartPage/CartPage.tsx [hunk 1]
```diff
@@ -72,6 +73,8 @@ export const CartPage = () => {
                 </ShopBreakdown>
               </Space>
 
+              {shopSession.experiments?.bundleDiscount && <BundleDiscountExtraProductLinks />}
```
Comment (guilhermespopolin) [inline apps/store/src/components/CartPage/CartPage.tsx:76]: Should we add a feature flag on our side for this? If I understood correctly, as long as storefront doesn't include experiments this feature will not be available but I just wonder if we want this sort of control on the FE as well 🤔

File: apps/store/src/features/bundleDiscount/useBundleDiscountExtraProductLinks.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    i18n: { language },
+  } = useTranslation()
+  const { shopSession } = useShopSession()
+  const productMetadata = useProductMetadata()
+  if (shopSession == null || productMetadata == null) return []
+  if (language !== 'se' && language !== 'se-en') {
+    console.warn(`Unexpected language: ${language}`)
+    return []
+  }
+  const productMetadataById = Object.fromEntries(productMetadata.map((item) => [item.id, item]))
```
Comment (guilhermespopolin) [inline apps/store/src/features/bundleDiscount/useBundleDiscountExtraProductLinks.ts:69]: NIce. I'd probably use reduce out of muscle memory and this is cleaner

**Review by guilhermespopolin — APPROVED** [review]
Nice. Looks promising 🙌🏻

File: apps/store/src/components/CartPage/CartPage.tsx (hunk 1 above)
Comment (alebedev) [reply apps/store/src/components/CartPage/CartPage.tsx:76]: Controlling it from storefront as single source of truth is easier to manage, so we don't need frontend flag

**Comment by alebedev** [comment]
### Merge activity

* **May 17, 4:25 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4278) rebased this pull request as part of a merge.

**Pull request merged**